msgcntr/messageforums-app/src/webapp/jsp/dfReviseForumSettingsAttach.jsp

```diff
@@ -87,6 +87,14 @@
     <h:form id="revise">
 		  <script>
             $(document).ready(function(){
+				// Improve accessibility in error messages.adding the error as title
+				var errorMessages = $('#revise\\:errorMessages');
+				if (errorMessages !== undefined) {
+					errorMessages.find('td').each(function() {
+						$(this).attr('title', $(this).html());
```

Comment:
This seems like something Sakai should be doing more regularly and would even go as far to say if there are any active sak-banner-error, sak-banner-warn, sak-banner-success should probably be doing this everywhere? If so should this go in headscript?

Reply:
Don't think so, this only affects the JSF tools that use the h:messages tag using the table layout, I think is fine resolving the issue here.